jwql/utils/utils.py

```diff
@@ -33,13 +33,108 @@
 import json
 import os
 import re
+import shutil
 
 from jwql.utils import permissions
 from jwql.utils.constants import FILE_SUFFIX_TYPES, JWST_INSTRUMENT_NAMES_SHORTHAND
 
 __location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
 
 
+def copy_files(files, out_dir):
```

Comment:
@bhilbert4 It would be useful to have an accompanying `test_copy_files` for this in `test_utils.py`, but we could handle that in a separate PR if you'd rather pass on it.

Reply:
Done. Test is passing.